addons/loyalty/models/loyalty_card.py

```diff
@@ -41,6 +41,7 @@ def _compute_display_name(self):
 
     use_count = fields.Integer(compute='_compute_use_count')
     active = fields.Boolean(default=True)
+    history_ids = fields.One2many(comodel_name='loyalty.history', inverse_name='card_id')
```

Comment:
this field should be readonly :thinking: 
because currently it is allowing to update history_line
![image](https://github.com/user-attachments/assets/61c1623b-e97c-42d5-a42a-f63f036fa576)